ehitchhiking-ui/src/app/shared/components/blacklist/blacklist.component.sass

```diff
@@ -0,0 +1,32 @@
+@import "../../../../styles/colour-constants"
+@import "../../../../styles/fonts-constants"
+@import "../../../../styles"
+
+.blacklist-title
+  text-align: center
+.blacklist
+  height: 20rem
+  overflow-y: scroll
+  overflow-x: hidden
+  margin: 0.5rem
+  .blacklist-user
+    border: 1px solid $black
+    display: flex
+    justify-content: space-around
+    align-items: center
+    .profile-photo
+      font-size: $display-2
+    .remove-icon
+      font-size: $title
+      display: table
+      vertical-align: middle
+      color: $dark-red
+    .blacklist-user-name
+      font-size: $button
+      margin-top: 1rem
+      width: 10rem
+      @extend .truncated
```

Comment:
pls remove @extend .truncated (@import "../../../../styles" too) and add truncated class in the html

Reply:
fixed